src/script/luna/mememu.cpp

```diff
@@ -1499,7 +1562,25 @@ class NPCMemory final : public NpcParent
             return obj->Reset[1] ? 0xFFFF : 0;
         else if(address == 0x128)
             return obj->Reset[2] ? 0xFFFF : 0;
-
+        else if(address >= 0x0A && address < 0x14) // Pinched
+        {
+            switch(address)
+            {
+            case 0x0A: // Pinched1
+                return valueToMem((int)obj->Pinched.Bottom1, ftype);
+            case 0x0C: // Pinched2
+                return valueToMem((int)obj->Pinched.Left2, ftype);
+            case 0x0E: // Pinched3
+                return valueToMem((int)obj->Pinched.Top3, ftype);
+            case 0x10: // Pinched4
+                return valueToMem((int)obj->Pinched.Right4, ftype);
+            case 0x12: // MovingPinched
+                return valueToMem((int)obj->Pinched.Moving, ftype);
+            default:
+                // use default handler
```

Comment:
I guess, the warning message should be added here in any cases of attempts to hack bytes in this area.

Reply:
Good idea, I'll fix that now in both cases